SparkPost/2web2ui
# AC-1976: Global tracking for all forms in suppressions and subaccounts

AC-1976: Global tracking for all forms in suppressions and subaccounts 

### What Changed

- Allowlisted forms from suppressions and subaccounts
- Added escaping for / in getFieldLabel
- Added cypress tests for subaccounts 

### How To Test

- Check if all the forms from suppressions and subaccounts are allowlisted
- Check if all sensitive fields from these forms are blocklisted

### To Do

- [ ] Add supporting cypress tests for suppressions ?
